libp2p/kademlia/protocol.py

```diff
@@ -11,15 +11,11 @@
 
 
 class KademliaProtocol(RPCProtocol):
-    """
-    There are four main RPCs in the Kademlia protocol
-    PING, STORE, FIND_NODE, FIND_VALUE
-    PING probes if a node is still online
-    STORE instructs a node to store (key, value)
-    FIND_NODE takes a 160-bit ID and gets back
-    (ip, udp_port, node_id) for k closest nodes to target
-    FIND_VALUE behaves like FIND_NODE unless a value is stored
-    """
+    """There are four main RPCs in the Kademlia protocol PING, STORE,
+    FIND_NODE, FIND_VALUE PING probes if a node is still online STORE instructs
```

Comment:
This docstring got messed up.  Looks like it should be manually re-formatted so that the definitions of each message type are done using a bulletted list.